clm_data/create_artifact.jl

```diff
@@ -12,20 +21,36 @@ if isdir(output_dir)
 else
     mkdir(output_dir)
 end
+for (file_path, file_url) in zip(FILE_PATHS, FILE_URLS)
+    println(file_url)
+    println(file_path)
+    if !isfile(file_path)
+        @info "$file_path not found, downloading it (might take a while)"
+        # The server has poor certificates, so we have to disable verification
+        downloader = Downloads.Downloader()
+        downloader.easy_hook =
+            (easy, info) -> Downloads.Curl.setopt(
+                easy,
+                Downloads.Curl.CURLOPT_SSL_VERIFYPEER,
+                false,
+            )
+        println(file_url)
+        println(file_path)
+        downloaded_file = Downloads.download(file_url; downloader)
+        Base.mv(downloaded_file, file_path)
+        Base.cp(file_path, joinpath(output_dir, basename(file_path)))
+    end
+end
+
+run(`python dominant_pft.py`)
+run(`python pft_variables.py`)
+run(`python dominant_mechanism.py`)
 
-if !isfile(FILE_PATH)
-    @info "$FILE_PATH not found, downloading it (might take a while)"
-    # The server has poor certificates, so we have to disable verification
-    downloader = Downloads.Downloader()
-    downloader.easy_hook =
-        (easy, info) -> Downloads.Curl.setopt(
-            easy,
-            Downloads.Curl.CURLOPT_SSL_VERIFYPEER,
-            false,
-        )
-    downloaded_file = Downloads.download(FILE_URL; downloader)
-    Base.mv(downloaded_file, FILE_PATH)
-    Base.cp(FILE_PATH, joinpath(output_dir, basename(FILE_PATH)))
+for output_file in OUTPUT_FILES
+    output_path = joinpath(output_dir, basename(output_file))
+    if !isfile(output_path)
+        Base.cp(output_file, output_path; force=true)
```

Comment:
Why is `force=true`?

Reply:
Sorry. I thought I removed that when I added the check to see if the file is already there. It is fixed now.